Extensions/super_markdown_editor.js

```diff
@@ -0,0 +1,262 @@
+//* TITLE Super Markdown Editor **//
+//* VERSION 1.0 REV A **//
+//* DESCRIPTION Add live previews and other functionality to the Markdown post editor **//
+//* DEVELOPER new-xkit **//
+//* FRAME false **//
+//* BETA false **//
```

Comment:
BETA doesn't need to be true for a first release. We didn't do that to most of the extensions we released, did we?